r2/r2/lib/jsontemplates.py

```diff
@@ -610,6 +610,7 @@ class LinkJsonTemplate(ThingJsonTemplate):
         gilded="gildings",
         hidden="hidden",
         is_self="is_self",
+        lang="lang",
```

Comment:
I can't believe I messed that up. Added to subreddit template and removed from other in 55adf2e.